helm/prometheus-rules/templates/platform/atlas/alerting-rules/logging-pipeline.rules.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: logging-pipeline.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+    - name: logging-pipeline
+      rules:
+        # Any alloy component that uses the loki.write component can throw such errors.
+        # This includes alloy-logs and the observability-gateway
+        - alert: LogForwardingErrors
+          annotations:
+            dashboard: 53c1ecddc3a1d5d4b8d6cd0c23676c31/alloy-logs-overview
+            description: '{{`More that 10% of the requests to Loki are failing.`}}'
+            opsrecipe: logging-pipeline/
+          expr: |-
+            (
+              100
+              *
+                (
+                    (
+                      sum by (cluster_id, installation, provider, pipeline, namespace, job, instance) (
+                        rate (
+                          loki_write_request_duration_seconds_count{status_code!~"2.."}[5m:]
+                        )
+                      )
+                    )
+                  /
+                    (
+                      sum by (cluster_id, installation, provider, pipeline, namespace, job, instance) (
+                        rate (
+                          loki_write_request_duration_seconds_count[5m:]
+                        )
+                      )
+                    )
+                )
+            )
+            > 10
+          for: 15m
+          labels:
+            area: platform
+            severity: page
+            team: atlas
+            topic: observability
+            cancel_if_cluster_status_creating: "true"
+            cancel_if_cluster_status_deleting: "true"
+            cancel_if_cluster_status_updating: "true"
+        # This alert pages when the loki source api component of the observability gateway is throwing errors
+        - alert: LogReceivingErrors
+          annotations:
+            dashboard: 53c1ecddc3a1d5d4b8d6cd0c23676c31/alloy-logs-overview
+            description: '{{`More that 10% of the loki requests to the observability gateway are failing.`}}'
+            opsrecipe: logging-pipeline/
+          expr: |-
+            (
+              100
+              *
+                (
+                    (
+                      sum by (cluster_id, installation, provider, pipeline, namespace, job, instance, route) (
+                        rate (
+                          loki_source_api_request_duration_seconds_count{route=~"(loki_)?api_v1_push", status_code!~"2.."}[5m:]
```

Comment:
Did you check these metrics exist?
On `golem`, observabilty-gateway is running for 17d and has no such metrics.

Reply:
They do, but they are only présent if the component receives metrics

Reply:
I pushed 3 lines to make you trust me :D ![image](https://github.com/user-attachments/assets/99c0bcf6-88a4-46d7-a93b-09f958400123)

Reply:
Fine with me then :grin: